Makefile

```diff
@@ -325,6 +330,9 @@ ADDON_CONTROLLER = $(PWD)/governance-policy-addon-controller
 
 .PHONY: kind-bootstrap-hosted
 kind-bootstrap-hosted: kind-install-hosted install-crds install-resources kind-deploy-cert-manager setup-managedcluster
+	@echo "Restarting propagator and addon-controller"
```

Comment:
I've started doing `# Restarting propagator and addon-controller` instead of `@echo`. Doesn't really matter, but I kind of like the free marker at the beginning and not needing to ignore the command.

Reply:
Apparently my editor doesn't know if that `#` really means a comment in a makefile. So I was worried I would break some syntax somewhere 😆